src/app/code/community/Omise/Gateway/Model/Payment/Offsitetesco.php

```diff
@@ -1,5 +1,5 @@
 <?php 
-class Omise_Gateway_Model_Payment_Offsitetesco extends Omise_Gateway_Model_Payment_BarcodeOffsite_Payment {
+class Omise_Gateway_Model_Payment_Offsitetesco extends Omise_Gateway_Model_Payment_Offline_Payment {
```

Comment:
Should it be called `Omise_Gateway_Model_Payment_Offsitetesco` and be in a file named `Offlinetesco.php`?

Reply:
The file naming also applies to the other models above